scripts/variantstore/wdl/GvsImportSample.wdl

```diff
@@ -0,0 +1,223 @@
+version 1.0 
+
+workflow GvsImportSample {
+
+  input {
+    File input_vcf
+    String external_sample_name
+    File interval_list
+    String gvs_sample_id
+    String project_id
+    String dataset_name
+    String? service_account_json_path
+    String? drop_state = "FORTY"
+    Boolean? drop_state_includes_greater_than = false
+    Boolean load_pet = false
+    Boolean load_ref_ranges = true
+    Boolean load_vet = true
+
+    Int? preemptible_tries
+    File? gatk_override = "gs://broad-dsp-spec-ops/scratch/andrea/gatk-ah-writeapi.jar"
+    String? docker
+  }
+
+  String docker_final = select_first([docker, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
+  
+  call CheckForDuplicateData {
+    input:
+      project_id = project_id,
+      dataset_name = dataset_name,
+      sample_name = external_sample_name,
+      gvs_sample_id = gvs_sample_id,
+      load_pet = load_pet,
+      load_ref_ranges = load_ref_ranges,
+      load_vet = load_vet,
+      service_account_json_path = service_account_json_path,
+      preemptible_tries = preemptible_tries
+  }
+
+  call ImportSample {
+    input:
+      input_vcf = input_vcf,
+      sample_name = external_sample_name,
+      gvs_sample_id = gvs_sample_id,
+      project_id = project_id,
+      dataset_name = dataset_name,
+      interval_list = interval_list,
+      service_account_json_path = service_account_json_path,
+      drop_state = drop_state,
+      drop_state_includes_greater_than = drop_state_includes_greater_than,
+      load_ref_ranges = load_ref_ranges,
+      load_pet = load_pet,
+      load_vet = load_vet,
+      gatk_override = gatk_override,
+      docker = docker_final,
+      preemptible_tries = preemptible_tries,
+      duplicate_check_passed = CheckForDuplicateData.done
+  }
+  
+  output {
+    Boolean loaded_in_gvs = true
+  }
+}
+
+
+task CheckForDuplicateData {
```

Comment:
In running, I noticed this step call-cached… Should it?  Should it be volatile instead?